docs/CODEOWNERS

```diff
@@ -0,0 +1,145 @@
+# Archipelago World Code Owners / Maintainers Document
+#
+# This file is used to notate the current "owners" or "maintainers" of any currently merged world folder. For any pull
+# requests that modify these worlds, a code owner must approve the PR in addition to a core maintainer. This is not to
+# be used for files/folders outside the /worlds folder, those will always need sign off from a core maintainer.
+#
+# All usernames must be GitHub usernames (and are case sensitive).
+
+# Adventure
+/worlds/adventure/ @JusticePS
+
+# A Link to the Past
+/worlds/alttp/ @Berserker66
+
+# ArchipIDLE
+/worlds/archipidle/ @LegendaryLinux
+
+# Sudoku (BK Sudoku)
+/worlds/bk_sudoku/ @Jarno458
+
+# Blasphemous
+/worlds/blasphemous/ @TRPG0 @BrandenEK
```

Comment:
Damocles just does client side stuff for Blasphemous so this should just be me actually.